conf/sample/bblayers-jetson-tx2.conf

```diff
@@ -0,0 +1,4 @@
+BBLAYERS += "\
+    ${BSPDIR}/sources/meta-tegra \
+    ${BSPDIR}/sources/meta-boot2qt/meta-tegra-extras \
```

Comment:
When meta-boot2qt is bumped to QtAS_5.14.0 (warrior) iof 5.13.1 (thud) the meta-xxx-extras BSP layers have been replaced with dynamic-layers in meta-boot2qt-distro.
But I assume this needs to be kept if meta-pelux shall be compatible with thud and previous Yocto releases.

Reply:
Yes you are correct.
I think it will be tricky to support both warrior and thud at the same time. At least for qt.
Also do we really wanna keep using meta-boot2qt-distro?
What parts of meta-tegra-extras are actually used?

Reply:
I would really like to see that we get rid of the dependency to meta-boot2qt-distro.
I have not investigated which part of the meta-xxx-extras that are used.
But will there be a problem if we keep the meta-xxx-extras layer in BBLAYER even if they don't exist? When we support zeus we can remove it, but we should strive to support at least 2 Yocto versions, w and w/o Qt.

Reply:
AFAIK bitbake will raise sanity check errors when parts of BBLAYER does not exist. Will do a test...